src/waterlinked_dvl/waterlinked_dvl_tcp.cpp

```diff
@@ -190,6 +192,18 @@ void WaterlinkedDvlTcp::f_parse_json_v2(Json::Value root) {
                 }
                 m_twist_publisher.publish(twist_msg);
             }
+            //Raw Twist
+            geometry_msgs::TwistWithCovarianceStamped twist_msg;
```

Comment:
raw twist needs to be moved before the velocity validation if statement. for both v2 and v3.

Reply:
Moved.